ansible.cfg

```diff
@@ -1,5 +1,5 @@
 [ssh_connection]
-scp_if_ssh = True
```

Comment:
it's an unrelated change, I accidentally pushed up, but I would very much like to address this so I don't accidentally push it up again.
As we discussed in Slack, I will get in the habit of creating draft PRs and/or using the `[WIP]` preface to the title to make it clear when it is ready for a review. Before it's ready, I will give myself a review so as to not waste your time.
Sorry again.

As far as this change, I think you know why I make it, but in case someone else doesn't or you have forgotten. I am unable to deploy to production instances with this change, which was made to get rid of a warning that was an annoyance for other devs. I briefly looked into it, but don't remember much about it and would prefer introducing this change in a separate PR that has an issue attached to it that explains it better.